### ISSUE TITLE
L2TP VPN crash on versions 4.0.1 and above

### ISSUE THREAD

**Issue by sinkuroi** [pwn20wndstuff/Undecimus#1765]
**Description**
On unc0ver versions 4.0.1 and above, L2TP VPN does not work. Error message claims that the server did not respond. Same server works fine in non-jailbroken state.

**To Reproduce**
1. Settings
2. VPN
3. Select L2TP VPN configuration
4. Set status to "on"

**Screenshots**
https://i.imgur.com/G5DlIQk.png

**Device:**
 - iOS Version: 13.3
 - iOS Device: iPhone 11
 - unc0ver Version: 4.0.3

**Place an "x" between the brackets if true:**
 - [x] this is a bug others will be able to reproduce
 - [x] this issue is present with all tweaks uninstalled(except for default packages) or disabled
 - [ ] this issue is present after a rootfs restore
 - [x] this issue is present on the latest version of unc0ver

Still occurs even with unc0ver option "Load Tweaks" disabled.
Another user on [Reddit](https://www.reddit.com/r/jailbreak/comments/f6hcjp/help_unc0ver_403_still_crashes_vpn_on_xs_max/?utm_source=share&utm_medium=web2x) is having the issue.

**Logs**
```{"blame":["\/usr\/lib\/substitute-inserter.dylib"],"app_name":"racoon","timestamp":"2020-02-20 11:02:28.37 +0000","app_version":"","slice_uuid":"63aa0053-4a85-36c4-aff4-87fc0c2bc2b7","build_version":"","share_with_app_devs":true,"is_first_party":true,"bug_type":"109","os_version":"iPhone OS 13.3 (17C54)","incident_id":"4805C018-5922-4D64-B592-CD0A108C43BA","name":"racoon","symbolicated":true}
Incident Identifier: 4805C018-5922-4D64-B592-CD0A108C43BA
CrashReporter Key:   d0e5f953d78fb6504c1b1457cd92ab6d3b578ea3
Hardware Model:      iPhone12,1
Process:             racoon [5379]
Path:                /usr/sbin/racoon
Identifier:          racoon
Version:             ???
Code Type:           ARM-64 (Native)
Role:                Unspecified
Parent Process:      launchd [1]
Coalition:           com.apple.racoon [902]


Date/Time:           2020-02-20 11:02:28.3120 +0000
Launch Time:         2020-02-20 11:02:28.2698 +0000
OS Version:          iPhone OS 13.3 (17C54)
Release Type:        User
Baseband Version:    1.03.12
Report Version:      104

Exception Type:  EXC_CRASH (SIGABRT)
Exception Codes: 0x0000000000000000, 0x0000000000000000
Exception Note:  EXC_CORPSE_NOTIFY
Triggered by Thread:  0

Thread 0 name:  Dispatch queue: com.apple.main-thread
Thread 0 Crashed:
0       libsystem_kernel.dylib        	0x1ae7d6efc 0x1ae7b1000 + 0x25efc
1       libsystem_pthread.dylib       	0x1ae6f68b8 0x1ae6f4000 + 0x28b8
2       libsystem_c.dylib             	0x1ae686a74 0x1ae610000 + 0x76a74
3     + substitute-inserter.dylib     	0x1001ec5e0 0x10012c000 + 0xc05e0
4       dyld                          	0x1003ef720 0x1003d8000 + 0x17720
5       dyld                          	0x1003efb08 0x1003d8000 + 0x17b08
6       dyld                          	0x1003ea460 0x1003d8000 + 0x12460
7       dyld                          	0x1003e9094 0x1003d8000 + 0x11094
8       dyld                          	0x1003e9160 0x1003d8000 + 0x11160
9       dyld                          	0x1003da4a8 0x1003d8000 + 0x24a8
10      dyld                          	0x1003df038 0x1003d8000 + 0x7038
11      dyld                          	0x1003d922c 0x1003d8000 + 0x122c
12      dyld                          	0x1003d9038 0x1003d8000 + 0x1038

Thread 0 crashed with ARM Thread State (64-bit):
    x0: 0x0000000000000000   x1: 0x0000000000000000   x2: 0x0000000000000000   x3: 0x0000000000000000
    x4: 0x0000000125d06930   x5: 0x0000000000000004   x6: 0x0000000000000000   x7: 0x00000000000000c5
    x8: 0x00000000000005b9   x9: 0xc3e7685fbc0ecdb9  x10: 0x00000000000f4070  x11: 0x0000000000179f41
   x12: 0x0000000125d04000  x13: 0x0000000000003fff  x14: 0x00000000b06e373c  x15: 0x0000000000002f55
   x16: 0x0000000000000148  x17: 0x00000001ece7ef10  x18: 0x0000000000000000  x19: 0x0000000000000006
   x20: 0x0000000000000303  x21: 0x0000000100449920  x22: 0x0000000100275000  x23: 0x000000016fd84ae0
   x24: 0x000000016fd84ad0  x25: 0x000000016fd85c70  x26: 0x000000016fd84ac0  x27: 0x000000016fd85c60
   x28: 0x000000016fd84b00   fp: 0x000000016fd84670   lr: 0x00000001ae6f68b8
    sp: 0x000000016fd84650   pc: 0x00000001ae7d6efc cpsr: 0x40000000
   esr: 0x56000080  Address size fault

Binary Images (dpkg):
0x100078000 - 0x100103fff + racoon arm64e  <63aa00534a8536c4aff487fc0c2bc2b7> /usr/sbin/racoon {"install_date":"2020-02-20 10:48:08 +0000","name":"iPhone Firmware (\/sbin)","identifier":"firmware-sbin","version":"0-1"}
0x10012c000 - 0x10024bfff + substitute-inserter.dylib arm64e  <20d28032cf87367389cf25c1d1399c66> /usr/lib/substitute-inserter.dylib {"install_date":"2020-02-20 10:48:04 +0000","name":"Substitute Support Package","identifier":"science.xnu.substituted","version":"0.1~b3"}

Binary Images (App Store):

Binary Images (Other):
0x1003d8000 - 0x10043ffff   dyld arm64e  <651eb4d8a0f03c97a0c46a8f6fc17a56> /usr/lib/dyld
0x1ae5c4000 - 0x1ae5dafff   libsystem_trace.dylib arm64e  <d8b2e2cdf750340388aba63c781581b9> /usr/lib/system/libsystem_trace.dylib
0x1ae5db000 - 0x1ae60efff   libxpc.dylib arm64e  <b7ae424e41d63a58a05cea0847830ddc> /usr/lib/system/libxpc.dylib
0x1ae60f000 - 0x1ae60ffff   libsystem_blocks.dylib arm64e  <298d0cb4af9b301ba31e0709413046f0> /usr/lib/system/libsystem_blocks.dylib
0x1ae610000 - 0x1ae68efff   libsystem_c.dylib arm64e  <4e543032e8f63ca69818e9f8fdb8f230> /usr/lib/system/libsystem_c.dylib
0x1ae68f000 - 0x1ae6cbfff   libdispatch.dylib arm64e  <2afaf8647f5c3011840db066e7377af1> /usr/lib/system/libdispatch.dylib
0x1ae6cc000 - 0x1ae6ecfff   libsystem_malloc.dylib arm64e  <0407f7fab6853a3a8db328f2eb85d4b3> /usr/lib/system/libsystem_malloc.dylib
0x1ae6ed000 - 0x1ae6f3fff   libsystem_platform.dylib arm64e  <cff9fe5cd8603000819ef5e15752dd46> /usr/lib/system/libsystem_platform.dylib
0x1ae6f4000 - 0x1ae6fefff   libsystem_pthread.dylib arm64e  <5f848a4c49073fc9a76d378fe8f2e4fc> /usr/lib/system/libsystem_pthread.dylib
0x1ae6ff000 - 0x1ae72ffff   libobjc.A.dylib arm64e  <a486c2b341b03f23af40c5579f5be97e> /usr/lib/libobjc.A.dylib
0x1ae730000 - 0x1ae79cfff   libcorecrypto.dylib arm64e  <7941b50b48d93654a5dfaf699a9db164> /usr/lib/system/libcorecrypto.dylib
0x1ae79d000 - 0x1ae7b0fff   libc++abi.dylib arm64e  <9c6349615c4b3df3a717d62877ab3a95> /usr/lib/libc++abi.dylib
0x1ae7b1000 - 0x1ae7dffff   libsystem_kernel.dylib arm64e  <4acde3c866303f879774d1741ea28f2b> /usr/lib/system/libsystem_kernel.dylib
0x1ae7e0000 - 0x1ae813fff   libdyld.dylib arm64e  <9059087b727b3b45b67209e5b03e6020> /usr/lib/system/libdyld.dylib
0x1ae814000 - 0x1ae81cfff   libsystem_darwin.dylib arm64e  <a799ce1b3d243b0980f1a5fcfc1c11da> /usr/lib/system/libsystem_darwin.dylib
0x1ae81d000 - 0x1ae877fff   libc++.1.dylib arm64e  <d9f3508f9f743705ac7c83efcf279139> /usr/lib/libc++.1.dylib
0x1ae878000 - 0x1ae8b9fff   libsystem_info.dylib arm64e  <aca5c5c953ed38b0a9c8cc5b7963fcd2> /usr/lib/system/libsystem_info.dylib
0x1ae8ba000 - 0x1aec37fff   CoreFoundation arm64e  <93bd750d2ece3e3884059e46efe482c6> /System/Library/Frameworks/CoreFoundation.framework/CoreFoundation
0x1aec38000 - 0x1aec9efff   SystemConfiguration arm64e  <2da2115e6cab3b86a21858d915b9d926> /System/Library/Frameworks/SystemConfiguration.framework/SystemConfiguration
0x1aec9f000 - 0x1aef6afff   Foundation arm64e  <80c31587f5383437966f300f6632eb95> /System/Library/Frameworks/Foundation.framework/Foundation
0x1aef6b000 - 0x1aef9dfff   libCRFSuite.dylib arm64e  <ed1305c6ef1f35f2ace7859d9f8685eb> /usr/lib/libCRFSuite.dylib
0x1aef9e000 - 0x1af121fff   CoreServices arm64e  <8bc685361a5238fb85dc333bf7273d14> /System/Library/Frameworks/CoreServices.framework/CoreServices
0x1af122000 - 0x1af184fff   libSparse.dylib arm64e  <c8f8efa09ead3905a54ca13f251b2ad9> /System/Library/Frameworks/Accelerate.framework/Frameworks/vecLib.framework/libSparse.dylib
0x1af185000 - 0x1af67efff   ImageIO arm64e  <afcf2555c1e13bed8bb4c78c1c35c44d> /System/Library/Frameworks/ImageIO.framework/ImageIO
0x1af81b000 - 0x1af95bfff   Security arm64e  <e5022c81c9a53c7cbfccaa299e595cf8> /System/Library/Frameworks/Security.framework/Security
0x1af95c000 - 0x1afa02fff   IOKit arm64e  <8410f83550363779bb03c4044be6dd4c> /System/Library/Frameworks/IOKit.framework/Versions/A/IOKit
0x1afa03000 - 0x1afa3afff   libMobileGestalt.dylib arm64e  <6d5ea0114512392789ff1d8c9102ebbe> /usr/lib/libMobileGestalt.dylib
0x1afaad000 - 0x1afd0dfff   libicucore.A.dylib arm64e  <184af8279281325fa56e5fc8fcceda19> /usr/lib/libicucore.A.dylib
0x1b0619000 - 0x1b079ffff   libsqlite3.dylib arm64e  <c8af1e21fa363921b2588002b98137c8> /usr/lib/libsqlite3.dylib
0x1b07a0000 - 0x1b07d2fff   MobileKeyBag arm64e  <2780973ce05d3566abfa5567d2973d90> /System/Library/PrivateFrameworks/MobileKeyBag.framework/MobileKeyBag
0x1b07d3000 - 0x1b07dcfff   libsystem_notify.dylib arm64e  <0db1f89f87a33c58b0eda2d0a7520b92> /usr/lib/system/libsystem_notify.dylib
0x1b0c45000 - 0x1b111ffff   libnetwork.dylib arm64e  <1c467a1610e133f29496a44b8ce7b1e6> /usr/lib/libnetwork.dylib
0x1b1231000 - 0x1b125bfff   CoreServicesStore arm64e  <181c7c7cfc4435468b91ce71ccbcc846> /System/Library/PrivateFrameworks/CoreServicesStore.framework/CoreServicesStore
0x1b1536000 - 0x1b154cfff   ProtocolBuffer arm64e  <cb77568d0c423ae1abe191b796d30031> /System/Library/PrivateFrameworks/ProtocolBuffer.framework/ProtocolBuffer
0x1b154d000 - 0x1b1567fff   CommonUtilities arm64e  <eeebcf1a184e3c2ebc876c3be5688046> /System/Library/PrivateFrameworks/CommonUtilities.framework/CommonUtilities
0x1b1568000 - 0x1b1568fff   libenergytrace.dylib arm64e  <78ff67d76a84315cbcd6571c5e4f770f> /usr/lib/libenergytrace.dylib
0x1b1569000 - 0x1b15a1fff   RunningBoardServices arm64e  <78fa48a7a1913166bad1176a5baa6b8d> /System/Library/PrivateFrameworks/RunningBoardServices.framework/RunningBoardServices
0x1b15a2000 - 0x1b161afff   BaseBoard arm64e  <8dc55bd22ef637bbb149baf3683341aa> /System/Library/PrivateFrameworks/BaseBoard.framework/BaseBoard
0x1b1c53000 - 0x1b1fb3fff   CFNetwork arm64e  <7052aa4b1c5b3b58b904ea280e0c8770> /System/Library/Frameworks/CFNetwork.framework/CFNetwork
0x1b31bc000 - 0x1b31cafff   AssertionServices arm64e  <7eaf805b7d6d34528ce455fff687077b> /System/Library/PrivateFrameworks/AssertionServices.framework/AssertionServices
0x1b31cb000 - 0x1b32a2fff   CoreTelephony arm64e  <9424072b2cbd3f94855c65a0760a9def> /System/Library/Frameworks/CoreTelephony.framework/CoreTelephony
0x1b32a3000 - 0x1b32a8fff   AggregateDictionary arm64e  <862910720b1434219b4bdab1c20743fe> /System/Library/PrivateFrameworks/AggregateDictionary.framework/AggregateDictionary
0x1b32a9000 - 0x1b32bffff   libsystem_asl.dylib arm64e  <df2c28764e223adea9710e7287f7e1f5> /usr/lib/system/libsystem_asl.dylib
0x1b39c7000 - 0x1b39d5fff   libsystem_networkextension.dylib arm64e  <0b5a7828c9b43739b7f69444cce391dc> /usr/lib/system/libsystem_networkextension.dylib
0x1b39d6000 - 0x1b39f6fff   CoreAnalytics arm64e  <98c3a047f2093ec3a4be99b8893bb633> /System/Library/PrivateFrameworks/CoreAnalytics.framework/CoreAnalytics
0x1b3c3a000 - 0x1b3d52fff   Network arm64e  <67df910684ec3c2c8c47c4a77a7c926b> /System/Library/Frameworks/Network.framework/Network
0x1b3db0000 - 0x1b3db7fff   libsystem_symptoms.dylib arm64e  <fb2a385ab40f318dbc3ead01f8984c3c> /usr/lib/system/libsystem_symptoms.dylib
0x1b4fb7000 - 0x1b4fc0fff   libsystem_containermanager.dylib arm64e  <3b43b4cbbbff37d5906d4267b3b23e33> /usr/lib/system/libsystem_containermanager.dylib
0x1b5863000 - 0x1b5dd6fff   CoreGraphics arm64e  <287f78e98c0638ee8e387be7c3a52c2b> /System/Library/Frameworks/CoreGraphics.framework/CoreGraphics
0x1b84bf000 - 0x1b84d0fff   IOSurface arm64e  <4a90a7a6f5a436a38ccc6439a21fc690> /System/Library/Frameworks/IOSurface.framework/IOSurface
0x1b8f05000 - 0x1b8f54fff   MobileWiFi arm64e  <25834dc235413fd5941afdaa7bf197cf> /System/Library/PrivateFrameworks/MobileWiFi.framework/MobileWiFi
0x1b8f86000 - 0x1b90d4fff   libFontParser.dylib arm64e  <99b7109e0ce73e4692f053347b8e9a59> /System/Library/PrivateFrameworks/FontServices.framework/libFontParser.dylib
0x1b9a21000 - 0x1b9caefff   vImage arm64e  <265fb9ae2db4350394da5fafaa95fb3b> /System/Library/Frameworks/Accelerate.framework/Frameworks/vImage.framework/vImage
0x1bc012000 - 0x1bc03cfff   PlugInKit arm64e  <4881eebc6b253c0285971bf9c72300d5> /System/Library/PrivateFrameworks/PlugInKit.framework/PlugInKit
0x1bd8c1000 - 0x1bd8c5fff   libsystem_configuration.dylib arm64e  <a92c46974d573124b9dd4097464668f7> /usr/lib/system/libsystem_configuration.dylib
0x1bdb26000 - 0x1bdb27fff   liblangid.dylib arm64e  <2a827b04feb03b17b4ad3ff7e9c95d19> /usr/lib/liblangid.dylib
0x1be923000 - 0x1be9affff   libTelephonyUtilDynamic.dylib arm64e  <a9a4e364599e33b79d747739c0b1c57a> /usr/lib/libTelephonyUtilDynamic.dylib
0x1befe4000 - 0x1bf0cefff   libxml2.2.dylib arm64e  <6f5daa11ee3134329a4e44340d060efc> /usr/lib/libxml2.2.dylib
0x1c11f3000 - 0x1c1419fff   NetworkExtension arm64e  <79cad53560553951a4167b93f34d4290> /System/Library/Frameworks/NetworkExtension.framework/NetworkExtension
0x1c1a58000 - 0x1c1a84fff   PersistentConnection arm64e  <981951176d9931a58b262a8b29bcf000> /System/Library/PrivateFrameworks/PersistentConnection.framework/PersistentConnection
0x1c2a2d000 - 0x1c2a2ffff   libsystem_sandbox.dylib arm64e  <384ec4785f773e19aff0ff8bfd7735f6> /usr/lib/system/libsystem_sandbox.dylib
0x1c2c37000 - 0x1c2c66fff   MobileInstallation arm64e  <a006311065723a959b2d5229400353b2> /System/Library/PrivateFrameworks/MobileInstallation.framework/MobileInstallation
0x1c2c67000 - 0x1c2d05fff   Metal arm64e  <e3af3fabbe9d370ba20197ba2e75ae54> /System/Library/Frameworks/Metal.framework/Metal
0x1c2d06000 - 0x1c2d0bfff   IOAccelerator arm64e  <89727af0d28938ff9e31b8b66ccbd246> /System/Library/PrivateFrameworks/IOAccelerator.framework/IOAccelerator
0x1c2d39000 - 0x1c2d40fff   libsystem_dnssd.dylib arm64e  <fbb48ad73dad3138bc4210c4fb391507> /usr/lib/system/libsystem_dnssd.dylib
0x1c34f4000 - 0x1c34f6fff   IOSurfaceAccelerator arm64e  <762b3a1e7b5c3fe7ae13a78a512d719d> /System/Library/PrivateFrameworks/IOSurfaceAccelerator.framework/IOSurfaceAccelerator
0x1c366b000 - 0x1c3681fff   libcoretls.dylib arm64e  <d18f15f0d39833159d79196c32f4a381> /usr/lib/libcoretls.dylib
0x1c36d9000 - 0x1c376cfff   libate.dylib arm64e  <f1b13c32ba0139de870ab42484212f1a> /usr/lib/libate.dylib
0x1c6422000 - 0x1c65f0fff   libBNNS.dylib arm64e  <16a98f6671d93444b9849888b6c85100> /System/Library/Frameworks/Accelerate.framework/Frameworks/vecLib.framework/libBNNS.dylib
0x1c78bd000 - 0x1c78c8fff   CaptiveNetwork arm64e  <8e04564059093e6da5c19f9b41080f31> /System/Library/PrivateFrameworks/CaptiveNetwork.framework/CaptiveNetwork
0x1c7a18000 - 0x1c7af2fff   libBLAS.dylib arm64e  <d8d889969b8031baad221c1837dbf4dc> /System/Library/Frameworks/Accelerate.framework/Frameworks/vecLib.framework/libBLAS.dylib
0x1cc57b000 - 0x1cc5abfff   EAP8021X arm64e  <d2df17b0c7d53332b693e6b0b77ed390> /System/Library/PrivateFrameworks/EAP8021X.framework/EAP8021X
0x1cef1b000 - 0x1cef8bfff   libarchive.2.dylib arm64e  <99bf48fde0043a67968ca13c2614a1fd> /usr/lib/libarchive.2.dylib
0x1cf088000 - 0x1cf089fff   libsystem_coreservices.dylib arm64e  <8f9cd0c29fa33f4f93893a2746f5229c> /usr/lib/system/libsystem_coreservices.dylib
0x1cf2ac000 - 0x1cf2b4fff   libcopyfile.dylib arm64e  <d1a467bab3163e5b9217227864727758> /usr/lib/system/libcopyfile.dylib
0x1d2f7d000 - 0x1d2f83fff   MobileSystemServices arm64e  <1c57d29a4d4a301084bfdf7aa93819d3> /System/Library/PrivateFrameworks/MobileSystemServices.framework/MobileSystemServices
0x1d44ec000 - 0x1d44fdfff   libSparseBLAS.dylib arm64e  <87fce2933cd037d3b81765856f66a4d9> /System/Library/Frameworks/Accelerate.framework/Frameworks/vecLib.framework/libSparseBLAS.dylib
0x1d5344000 - 0x1d5358fff   libLinearAlgebra.dylib arm64e  <b7392f4a63663967a1bd44a8a878775e> /System/Library/Frameworks/Accelerate.framework/Frameworks/vecLib.framework/libLinearAlgebra.dylib
0x1d717d000 - 0x1d71b7fff   libpcap.A.dylib arm64e  <08eafa0b66f739dda5876a23d500ce17> /usr/lib/libpcap.A.dylib
0x1d7510000 - 0x1d75b6fff   libvDSP.dylib arm64e  <9f1b4e370a99382a8d89b5af0ea6415b> /System/Library/Frameworks/Accelerate.framework/Frameworks/vecLib.framework/libvDSP.dylib
0x1d7f13000 - 0x1d7f13fff   Accelerate arm64e  <daa9c13c7fdb38a2b5a4e6f3df7659d6> /System/Library/Frameworks/Accelerate.framework/Accelerate
0x1d7f15000 - 0x1d8252fff   libLAPACK.dylib arm64e  <987217b663813be989eb40f1df2d4ff1> /System/Library/Frameworks/Accelerate.framework/Frameworks/vecLib.framework/libLAPACK.dylib
0x1d8253000 - 0x1d8257fff   libQuadrature.dylib arm64e  <288ce298873c38cc90334401bcf01f58> /System/Library/Frameworks/Accelerate.framework/Frameworks/vecLib.framework/libQuadrature.dylib
0x1d8258000 - 0x1d82b1fff   libvMisc.dylib arm64e  <2accc971390233078f050720dc2e1ceb> /System/Library/Frameworks/Accelerate.framework/Frameworks/vecLib.framework/libvMisc.dylib
0x1d82b2000 - 0x1d82b2fff   vecLib arm64e  <e898be21a999319ba485548bb4859e6b> /System/Library/Frameworks/Accelerate.framework/Frameworks/vecLib.framework/vecLib
0x1d897a000 - 0x1d897afff   MobileCoreServices arm64e  <4bf207f6d9dc3564b639aec23c31e5fd> /System/Library/Frameworks/MobileCoreServices.framework/MobileCoreServices
0x1d8987000 - 0x1d898dfff   libCoreFSCache.dylib arm64e  <b53da010d4e437b99dd6050c27c6aa41> /System/Library/Frameworks/OpenGLES.framework/libCoreFSCache.dylib
0x1d9333000 - 0x1d93c4fff   APFS arm64e  <20e1ee3664453bc5a59b27f41d696d2b> /System/Library/PrivateFrameworks/APFS.framework/APFS
0x1d9eec000 - 0x1d9f2efff   AppleJPEG arm64e  <6d3e92c57f903f35b4b5a50b1a20373d> /System/Library/PrivateFrameworks/AppleJPEG.framework/AppleJPEG
0x1d9f62000 - 0x1d9f85fff   AppleSauce arm64e  <8897e07e5b8f329fa82cf5a8583f4bb3> /System/Library/PrivateFrameworks/AppleSauce.framework/AppleSauce
0x1da18a000 - 0x1da1bafff   Bom arm64e  <568b245daaa931d28790260b47c0f989> /System/Library/PrivateFrameworks/Bom.framework/Bom
0x1dc261000 - 0x1dc26dfff   libhvf.dylib arm64e  <071f10832d383b76afdbd3da221e72e7> /System/Library/PrivateFrameworks/FontServices.framework/libhvf.dylib
0x1e36fa000 - 0x1e36fbfff   libSystem.B.dylib arm64e  <7f892924171838d0870ee661d1aa4a22> /usr/lib/libSystem.B.dylib
0x1e3805000 - 0x1e381afff   libapple_nghttp2.dylib arm64e  <ba029feafe2739cb857c3dcaea611d6c> /usr/lib/libapple_nghttp2.dylib
0x1e3893000 - 0x1e38a3fff   libbsm.0.dylib arm64e  <d9098218c078393a9e586b694e1c9e23> /usr/lib/libbsm.0.dylib
0x1e38a4000 - 0x1e38b0fff   libbz2.1.0.dylib arm64e  <c5cfe7d65aa83c0289e60410a4998d51> /usr/lib/libbz2.1.0.dylib
0x1e38b1000 - 0x1e38b1fff   libcharset.1.dylib arm64e  <30c87f2f727b3e81b7c3300b0ae4a76f> /usr/lib/libcharset.1.dylib
0x1e38c4000 - 0x1e38dbfff   libcompression.dylib arm64e  <055bea1c1fb03f74891a55379b232122> /usr/lib/libcompression.dylib
0x1e38dc000 - 0x1e38ddfff   libcoretls_cfhelpers.dylib arm64e  <adfa5c10b2d434a499d34ea1a2735c1d> /usr/lib/libcoretls_cfhelpers.dylib
0x1e38de000 - 0x1e38e4fff   libcupolicy.dylib arm64e  <d3e787ff65693dfaaa0c3e3a8e2843b1> /usr/lib/libcupolicy.dylib
0x1e396c000 - 0x1e3a5efff   libiconv.2.dylib arm64e  <04ed23c647eb330b8e6a8993a451bd07> /usr/lib/libiconv.2.dylib
0x1e3a6e000 - 0x1e3a72fff   libipsec.A.dylib arm64e  <66f4e4b2cb14331585fb5c5f6f8f6277> /usr/lib/libipsec.A.dylib
0x1e3a80000 - 0x1e3a98fff   liblzma.5.dylib arm64e  <4c28b90255ca39c6adbf0ddde0abac6d> /usr/lib/liblzma.5.dylib
0x1e3e4f000 - 0x1e3e63fff   libnetworkextension.dylib arm64e  <7277e8a316233035becb0c3d1ff4eb5f> /usr/lib/libnetworkextension.dylib
0x1e41ef000 - 0x1e4207fff   libresolv.9.dylib arm64e  <25963e1e577034849ed4e2f593d3a7e4> /usr/lib/libresolv.9.dylib
0x1e424a000 - 0x1e424dfff   libutil.dylib arm64e  <0b04f33db5e934a78f9a8c1ba55565c0> /usr/lib/libutil.dylib
0x1e427b000 - 0x1e428cfff   libz.1.dylib arm64e  <fafaa0b435f730aa87616a05ec84b8cd> /usr/lib/libz.1.dylib
0x1e46cc000 - 0x1e46d1fff   libcache.dylib arm64e  <71da2360617e3daf92b1cb616e7cf6c0> /usr/lib/system/libcache.dylib
0x1e46d2000 - 0x1e46e2fff   libcommonCrypto.dylib arm64e  <a6eb95d01dd33d93ab7d2cc2c5f1432d> /usr/lib/system/libcommonCrypto.dylib
0x1e46e3000 - 0x1e46e6fff   libcompiler_rt.dylib arm64e  <3fef8706ff563e8196c2c0e987108915> /usr/lib/system/libcompiler_rt.dylib
0x1e47bd000 - 0x1e47bdfff   liblaunch.dylib arm64e  <8d8e0d74131d3353a96b2e382bd25d6a> /usr/lib/system/liblaunch.dylib
0x1e47be000 - 0x1e47c3fff   libmacho.dylib arm64e  <d57ecc933d383549b596a37f3aa5f6e5> /usr/lib/system/libmacho.dylib
0x1e47c4000 - 0x1e47c5fff   libremovefile.dylib arm64e  <532f053ab81634fa9785432a44ffec51> /usr/lib/system/libremovefile.dylib
0x1e47c6000 - 0x1e47c7fff   libsystem_featureflags.dylib arm64e  <8da1b6132fec3a1182233b68b5d13a59> /usr/lib/system/libsystem_featureflags.dylib.
0x1e47f6000 - 0x1e47fbfff   libunwind.dylib arm64e  <e0ee449ef786342596215cc058a119af> /usr/lib/system/libunwind.dylib```

**Comment by annrein** [comment]
I also cannot connect VPN with Unc0ver higher than 4.0.0, in my case it is IPsec.

**Comment by mingoo0402** [comment]
I had same problem during using L2TP. When I reboot and return to unjailbreaken state, VPN works normlally.  Even after updating to Unc0ver ver 4.1.1, the bug still persists.

**Comment by YtnbFirewings** [comment]
I had same problem during using IPSEC. iPhone XS Max
Unc0ver ver 4.0.3.
I can not jailbreak my device by use new unc0ver version（4.1.1 and above）.

**Comment by annrein** [comment]
VPN problems still exist in Unc0ver 4.2.1. Do you need crash logs?

**Comment by gkmasta** [comment]
The VPN issue seems to be resolved for me with unc0ver 4.3.1.
Before I was only able to establish a VPN connection using a third party app, but now my iPhone L2TP VPN profile seems to connect flawlessly. Thanks!
